app/views/govuk_publishing_components/components/docs/chat_entry.yml

```diff
@@ -0,0 +1,32 @@
+name: Chat entry
+description: Entry point for GOV.UK Chat
+accessibility_criteria: |
+  The heading must:
+
+  - be part of a correct heading structure for a page
+  - be semantically represented as a heading
+  - convey the heading level
+shared_accessibility_criteria:
+  - link
+examples:
+  default:
+    data:
+  with_custom_href,_heading_and_or_description:
+    data:
+      href: /chat
```

Comment:
Nitpick: would it be clearer to use something other than the default here?

Reply:
The default will likely be the most common way that the component will be used. Perhaps I should add a description for it for clarity?

Reply:
Sure, the default is likely to be unchanged, but to demonstrate that a different URL can be passed I think it's worth demonstrating that.

Related - the test that checks that a custom URL can be passed also uses the default, so it doesn't actually test that a different URL can be passed.

Reply:
Cool thanks, I've updated the default to include the `heading_text` option and also modified the test to use a different `href` instead of the default one.